src/main/java/course_project/sea_battle/Game.java

```diff
@@ -0,0 +1,79 @@
+package course_project.sea_battle;
+
+import course_project.sea_battle.boards.MyBoard;
+import course_project.sea_battle.boards.MyShots;
+import course_project.sea_battle.model.Player;
+import course_project.sea_battle.model.Ship;
+import course_project.sea_battle.service.inputs.InputShipReader;
+import course_project.sea_battle.service.NameSetter;
+import course_project.sea_battle.service.inputs.InputShooterReader;
+import course_project.sea_battle.service.ShipPlacer;
+
+import java.util.Scanner;
+
+public class Game extends Thread {
```

Comment:
The fields look good, except that indentation is shifted 
![image](https://user-images.githubusercontent.com/58872509/134216618-444620c5-8aff-42a8-b3ad-1b0a2fa6c499.png)

Reply:
Also, please introduce automatic ship placement

Reply:
Good job! Nice interface, well-chosen abstractions, architecture also looks fine. Good skills in Java Core and Java libraries. Special thanks for the description file and png, it helps. 

To make better(opt): introduce automatic ship placement and PvE gameplay. 

Approved!

Reply:
![2021-09-22_10-03-10](https://user-images.githubusercontent.com/75118781/134297886-24ad037a-ad24-415b-99f8-c125835d97cb.png)

I don't see any shifting. Maybe the reason is that I have windows, I don't know. How did you get in? In gameplay or in tests?

Reply:
Automatic ship placement: Done!

Reply:
Player vs Computer(easy) mode: Done